specification/paths/Returns-v1-shops-shop_id-returns-return_id-approve.json

```diff
@@ -0,0 +1,24 @@
+{
+  "post": {
+    "tags": [
+      "Returns"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "returns.manage"
+        ]
+      }
+    ],
+    "summary": "Approve a return.",
+    "description": "This endpoint modifies the status of a return resource to `return-approved`",
+    "responses": {
+      "204": {
+        "description": "The return is approved."
+      },
+      "404": {
+        "description": "No return was found for the given id."
+      }
```

Comment:
We can introduce a 422 in situations when the transition is forbidden because for example the return is already approved (it could be that there are more state transitions that are forbidden)